app/locale/en.coffee

```diff
@@ -1670,7 +1670,7 @@ module.exports = nativeDescription: "English", englishDescription: "English", tr
     certificate_btn_print: "Print"
     certificate_btn_toggle: "Toggle"
     ask_next_course: "Want to play more? Ask your teacher for access to the next course."
-    set_start_locked_level: "Assign up to level"
+    set_start_locked_level: "Lock levels starting at"
```

Comment:
Should we add the `# {change}` comment so we can propagate that this line was updated.